docker/runserver.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Create the local_settings.py to connect to the docker mysql
+echo "DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.mysql',
+        'NAME': 'msr14',
+        'USER': 'msr14',
+        'PASSWORD': 'msr14',
+        'HOST': 'database',
+        'PORT': '3306'}}" > repo_health/local_settings.py
```

Comment:
We will need to set DEBUG = True in the local settings so the Django app runs in a production mode.   This may be easier to set in the settings file.